cpg_workflows/large_cohort/relatedness.py

```diff
@@ -83,9 +83,16 @@ def flag_related(
 ) -> hl.Table:
     """
     Rank samples and flag samples to drop so there is only one sample per family
-    left, with the highest rank in the family.
-
-    `sample_qc_ht` has to have a `filters` and `var_data_chr20_mean_dp` columns.
+    left, with the highest rank in the family. The ranking is based on either
+    'var_data_chr20_mean_dp' or 'autosomal_mean_dp' depending on the
```

Comment:
Nice that the introspection is as tidy as `'foo' in ht.row`! Probably should update this comment to say “depending on which is present in the table” or so.

Reply:
Good call, I've changed it to the following:
```test
Rank samples and flag samples to drop so there is only one sample per family
left, with the highest rank in the family. The ranking is based on either
'var_data_chr20_mean_dp' or 'autosomal_mean_dp' depending on which is present
in the `sample_qc_ht` table. The presence of either column is determined by
the 'inf_ploidy_using_var' parameter in the config toml.
```